### DISCUSSION TITLE
Order by field on relational table

### DISCUSSION THREAD

**Discussion by webnoob** [prisma/prisma#3005] [Q&A] [answered]
Is it possible to do something like below?

```sql
select
distinct m.userId, m.id
from messagethread as m
inner join messagepost as mp on m.id = mp.threadId
order by 
mp.createdDate asc; // This part
```

I know we can order by but I'm looking to order the result set based on a relational field.

**Comment by ryands17** [comment, marked as answer]
Hey @webnoob :wave: 
Unfortunately a raw query like the one above is the only way to order by relations. There is an open request [here](https://github.com/prisma/prisma-client-js/issues/249) that you can add a :+1: on so that we can prioritize the request.

**Reply by webnoob** [reply]
Ah thanks, I did search for something before posting. Thanks for the link.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
